dev_missing_price/dev_missing_price.py

```diff
@@ -0,0 +1,59 @@
+
+from woocommerce import API
+import os
+import csv
+
+woo_api = API(
+    url=os.environ.get('url'),
```

Comment:
If the environment variables are not set then this will get `None` and it will error. Then you have to do more work to try to figure out what is wrong.
So it is better to get the env variables first and check if they exist or not. If they dont exist throw an error that is clear.
That would make the script much more user friendly.